### PR TITLE
Fix/cm-418/cm-417

### PR DESCRIPTION
**Pull request by mgcouchbase**
#### Fixes #.

- [ yes] Ran `flake8`
- [ yes ] Ran `run_repo_tests.sh`
fixing the  .net stop service issue and  memory flush issue

#### Changes proposed in this pull request:

-  http://uberjenkins.sc.couchbase.com:8080/job/CBLITE_Net-windows-TestSever-Query-tests/255/

After fixing both issues we got clean run 
http://uberjenkins.sc.couchbase.com:8080/job/CBLITE_Net-windows-TestSever-Query-tests/259/
-

### CODE REVIEW

File: libraries/provision/ansible/playbooks/stop-testserver-windows.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-      win_file:
-        path: C:\PROGRA~1\dotnet\TextLogging
-        state: absent
+    - debug: msg="Stoping the Testserver "
+    - debug: msg="Copying logs at - {{ log_full_path }}"
+    - name: Stop TestServer
+      win_nssm:
+        name: TestServer
+        application: C:\PROGRA~1\dotnet\dotnet.exe
+        state: stopped
```
Comment (euniceatcouchbase) [inline libraries/provision/ansible/playbooks/stop-testserver-windows.yml:13]: why do you remove the log file clean up step? is the clean up step no longer needed?

File: CBLClient/Query.py [hunk 2]
```diff
@@ -427,11 +427,11 @@ def query_fts_with_ranking(self, database, prop, val,
...
-        self._client.release(obj)
-
     def query_arthimetic(self, database):
         args = Args()
         args.setMemoryPointer("database", database)
 
         return self._client.invokeMethod("query_arthimetic", args)
+
+    def release(self, obj):
+        self._client.release(obj)
```
[the remark this answers is not in the document]
Comment (sridevi-15) [reply CBLClient/Query.py:437]: yeah same question

File: libraries/provision/ansible/playbooks/stop-testserver-windows.yml (hunk 1 above)
Comment (sridevi-15) [reply libraries/provision/ansible/playbooks/stop-testserver-windows.yml:13]: @euniceatcouchbase , we had a discussion and found that stop-testserver-windows is supposed to stop the Testserver and fetch logs, but When Manasa showed me the script, there is no stop for testServer , but deleting logs. I asked her to replace with stop TestServer and we do not want to remove the logs as in Teardown, we fetch the logs if there is failure. Also this stop-testserver can be used in the test stop and restart and we should not delete the logs during the test when we stop the testserver. 

AFter adding custom-logging feature, there are no logs getting generated under C:\PROGRA~1\dotnet\TextLogging. that is the reason failing, so it does not make send to try to remove log which does not exist

Comment (euniceatcouchbase) [reply libraries/provision/ansible/playbooks/stop-testserver-windows.yml:13]: got it, thanks for the explanation.

File: CBLClient/Query.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mgcouchbase) [reply CBLClient/Query.py:437]: Looks like order execution matters. I tried without this 10 times and test failed , after this change tests passing consistently .

[the remark this answers is not in the document]
Comment (euniceatcouchbase) [reply CBLClient/Query.py:437]: interesting. it's better to find out how this release function is referenced, https://stackoverflow.com/questions/44423786/does-the-order-of-functions-in-a-python-script-matter

[the remark this answers is not in the document]
Comment (sridevi-15) [reply CBLClient/Query.py:437]: interesting to me, so far it should not matter. Looks like something is causing. Anyways for now we can go with this

**Pull request merged**